### DISCUSSION TITLE
ValueError: The test function test_alive is using the auto_container fixture but the parent module is not setting the 'CONTAINER_IMAGES' variable

### DISCUSSION THREAD

**Discussion by lmilbaum** [dcermak/pytest_container#196] [Q&A]
I'm still a newbie with this plugin. https://github.com/redhat-et/locallm/pull/96 is my initial contribution. Can't figure out why pytest throws the error https://github.com/redhat-et/locallm/pull/96#issuecomment-2016947305

**Comment by dcermak** [comment]
You have to define the variable `CONTAINER_IMAGES` in the test file, not in `conftest.py`.
